internal/cli/atlas/networking/peering/create/azure.go

```diff
@@ -70,7 +72,7 @@ func (opts *AzureOpts) containerExists() (*atlas.Container, error) {
 		return nil, err
 	}
 	for i := range r {
-		if r[i].Region == opts.region {
+		if strings.EqualFold(r[i].Region, opts.region) {
```

Comment:
rather than doing this we should do a normalization of the input, ie, we uppercase the input, which is a once off operation, see my comment in the aws command

Reply:
also wath out here with normalization as for example
Azure: centralus
Atlas: US_CENTRAL